medic_analysis/scripts/tSNR_processing.py

```diff
@@ -83,7 +84,13 @@ def main():
                     print(f"Submitting Job: {subject_dir.name}, {session_name}, {run_dir.name}")
                     futures.append(
                         executor.submit(
-                            compute_tSNR, run_dir, pipeline, subject_dir.name, session_name, run_dir.name, wmparc_path
+                            compute_tSNR,
+                            run_dir,
+                            pipeline,
+                            subject_dir.name,
+                            session_name,
+                            run_dir.name,
+                            wmparc_path,
```

Comment:
This variable is undefined.

Reply:
Addressed in https://github.com/tsalo/medic_analysis/pull/1